packages/terra-dialog-modal/src/DialogModal.jsx

```diff
@@ -63,81 +51,45 @@ const propTypes = {
 const defaultProps = {
   children: null,
   isOpen: false,
-  isFocused: false,
-  releaseFocus: null,
-  requestFocus: null,
   width: '1120',
 };
 
-class DialogModal extends React.Component {
-  componentDidMount() {
-    if (this.props.isFocused && this.props.isOpen && this.props.requestFocus) {
-      this.props.requestFocus();
-    }
-  }
+const DialogModal = (props) => {
+  const {
+    header,
+    footer,
+    children,
+    onRequestClose,
+    isOpen,
+    ariaLabel,
+    width,
+    ...customProps
+  } = props;
 
-  componentDidUpdate() {
-    if (this.props.isOpen) {
-      if (this.props.requestFocus) {
-        this.props.requestFocus();
-      }
-    } else if (this.props.releaseFocus) {
-      this.props.releaseFocus();
-    }
+  if (!props.isOpen) {
```

Comment:
Should be able to use the isOpen you've extracted from props here.

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/608/commits/5ee4e0544b820356cb994d4c507499efa1fa9be7